### ISSUE TITLE
Question: how to set up Lando for a Build Tools project?

### ISSUE THREAD

**Issue by nelsonamaya82** [pantheon-systems/example-wordpress-composer#197]
After running the `lando init` command with the Pantheon recipe, the .gitignore file of this template is rewritten by the .gitignore file from Pantheon, loosing some of the rules we have here like `/vendor/`, among others.  Do you know how can we avoid this?

Thank you.

**Comment by ataylorme** [comment]
@nelsonamaya82 my guess is `lando init` copies code from the Pantheon site down locally.

tl;dr I only use Lando to facilitate creating the correct Docker containers, wiring them up, and (occasionally) fetching the database/media files from Pantheon

The workflow with Lando and Build Tools that has worked for me is:

- Clone from GitHub (or GitLab or BitBucket) locally
- Manually create `.lando.yml`
- When doing `lando pull` use `--code=none` so that only the database and media files are pulled from Pantheon
  - `lando pull` by default copies code, database and media files from Pantheon to your local machine when we want to only interact with code on GitHub in a Build Tools workflow
- Use `git push` normally, like I would without Lando, and do not use `lando push`
  - `lando push` sends code to Pantheon when we want it to be sent to GitHub in a Build Tools workflow

After initial setup my workflow looks like:
- Do not interact with Pantheon directly
- Use `git` locally to create a new feature branch and push to GitHub, opening a pull request (CI creates the Pantheon multidev and deploys code)
- Use `lando pull --code=none` when my local database/media files become to stale to get the latest copies from the site on Pantheon

**Comment by nelsonamaya82** [comment]
Thanks for your answer @ataylorme.  In that case I think the [Working locally with Lando](https://github.com/pantheon-systems/example-wordpress-composer#working-locally-with-lando) section on the readme should be updated with the workflow you described above, instead of suggesting the use of the Pantheon recipe as it will rewrite the .gitignore and possibly some other files, do you agree?  If so I can create a PR with this update.

**Comment by ataylorme** [comment]
@nelsonamaya82 yes, I agree. I didn't realize `lando init` overwrote things during its initial setup.

I would happily review a pull request.
